test/schema/ListPublicEvents.schema.js

```diff
@@ -0,0 +1,36 @@
+const listPublicEventsSchema = {
+  type: 'array',
+  items: {
+    type: 'object',
+    properties: {
+      id: { type: 'string' },
+      type: { type: 'string' },
+      actor: {
+        type: 'object',
+        properties: {
+          id: { type: 'integer' },
+          login: { type: 'string' },
+          display_login: { type: 'string' },
+          gravatar_id: { type: 'string' },
+          url: { type: 'string' },
+          avatar_url: { type: 'string' }
+        }
+      },
+      repo: {
+        type: 'object',
+        properties: {
+          id: { type: 'integer' },
+          name: { type: 'string' },
+          url: { type: 'string' }
+        }
+      },
+      payload: {
+        type: 'object'
```

Comment:
choose an event type, and put the properties of that event type in this payload structure